extensions/data-transfer/portability-data-transfer-microsoft/src/test/java/org/dataportabilityproject/transfer/microsoft/transformer/calendar/ToCalendarModelTransformerTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 The Data-Portability Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataportabilityproject.transfer.microsoft.transformer.calendar;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.dataportabilityproject.transfer.microsoft.helper.TestTransformerContext;
+import org.dataportabilityproject.transfer.microsoft.transformer.TransformerContext;
+import org.dataportabilityproject.types.transfer.models.calendar.CalendarModel;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class ToCalendarModelTransformerTest {
+    private ToCalendarModelTransformer transformer;
+    private ObjectMapper mapper;
+    private TransformerContext context;
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testTransform() throws IOException {
+        Map<String, Object> rawEvent = mapper.readValue(JSON, Map.class);
+
+        CalendarModel calendar = transformer.apply(rawEvent, context);
+        
+        Assert.assertEquals("123", calendar.getId());
+        Assert.assertEquals("Calendar", calendar.getName());
+        Assert.assertEquals("Calendar", calendar.getDescription());
+    }
+
+    @Before
+    public void setUp() {
+        transformer = new ToCalendarModelTransformer();
+        mapper = new ObjectMapper();
+        context = new TestTransformerContext();
+    }
+
+    private static final String JSON = "{\n" +
```

Comment:
Can we change the name to something more descriptive, e.g. SAMPLE_CALENDAR_EVENT or add a comment to describe

Reply:
Sure